models/query/report-query.js

```diff
@@ -68,15 +69,19 @@ ReportQuery.prototype.toMongooseFilter = function() {
   // Determine author filter
   if (this.author) {
     filter.author = {};
-    filter.author.$in = toRegexp.alli(this.author.trim().split(/\s*,\s*/).sort());
+    filter.author.$options = 'i';
+    filter.author.$regex = this.author;
```

Comment:
Would love a basic explanation of what this did before and what it does now. I'm curious why they were all turned into an array before if it's as simple as using $regex instead. Does this mean we can only search one author at a time now?

Reply:
Previously we used a handwritten method of regexing the inputs. However, mongoDB now supports built in regex functions, so this is unecessary. Somewhere along the way, this previous method stopped working very well.

Reply:
Also yes, we can only search one author at a time.